src/engine/qcommon/common.cpp

```diff
@@ -910,18 +910,36 @@ void Com_Frame()
 		}
 		else
 		{
-			if ( com_minimized->integer && maxfpsMinimized.Get() > 0 )
+			int max;
+
+			if ( com_minimized->integer )
+			{
+				max = maxfpsMinimized.Get();
+			}
+			else if ( com_unfocused->integer )
 			{
-				minMsec = 1000 / maxfpsMinimized.Get();
+				max = maxfpsUnfocused.Get();
 			}
-			else if ( com_unfocused->integer && maxfpsUnfocused.Get() > 0 )
+			else
+			{
+				max = maxfps.Get();
+			}
+
+			// A positive maxfps caps the fps to the given number, with an implicit
+			// cap at 333fps to avoid bugs. Above 333fps minMsec is less than 3.
+			// At 1 or 2 minMsec, the game still runs but exhibits various issues
+			// such as first-person weapon model flickering, or client having
+			// connection issues with server.
+			if (max > 0)
 			{
-				minMsec = 1000 / maxfpsUnfocused.Get();
+				minMsec = std::min(1000/max, 3);
```

Comment:
The previous code used spaces between division operators:
```suggestion
				minMsec = std::min(1000 / max, 3);
```

Reply:
Right.